### PR TITLE
Removed [-Inf,Inf] from validator translations.

### PR DESCRIPTION
**Pull request by maryo**
Hi. I am validating the forms using https://github.com/formapro/JsFormValidatorBundle
But the bundle apparently does not support transChoice so the error messages shown by the JS contain `[-Inf,Inf]` string. It could be fixed in that bundle OFC (actually it could not be so simple to determine the count value of every constraint) but I guess the interval is redundant when there is no choice thus it can be ommited. Or am I wrong? Is there a reason I am not aware of? Thanks.

EDIT: Well. After thinking about it I fixed it in my fork of the mentioned bundle to use transChoice with zero count for extracting messages instead. But I still think it could be merged.

### CODE REVIEW

File: Resources/translations/validators.cs.yml [hunk 1]
```diff
@@ -2,24 +2,24 @@ fos_user:
...
-        short: '[-Inf,Inf]Nové heslo je příliš krátké'
+        short: 'Nové heslo je příliš krátké'
     current_password:
         invalid: 'Nesprávné heslo'
     group:
         blank: 'Vyplňte název skupiny'
-        short: '[-Inf,Inf]Název skupiny je příliš krátký'
-        long: '[-Inf,Inf]Název skupiny je příliš dlouohý'
+        short: 'Název skupiny je příliš krátký'
+        long: 'Název skupiny je příliš dlouhý'
```
Comment (maryo) [inline Resources/translations/validators.cs.yml]: fixed typo

**Comment by XWB** [comment]
@OskarStark What is your opinion on this?

**Comment by OskarStark** [comment]
IMO `[-Inf,Inf]` is useless, because it is the same like removing it completely, or am I wrong?

**Comment by XWB** [comment]
Thanks.

**Pull request merged**